profiles/rakesh9100.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/73993775/212983418-aaad5362-7792-4125-8faa-c0a965eb3572.png",
+    "issueId": 1665,
+    "name": "Rakesh Roshan",
+    "username": "rakesh9100"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Done 😌